.github/workflows/deploy.yml

```diff
@@ -0,0 +1,64 @@
+name: Deploy to TestFlight
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Number version to deploy'
+        required: true
+
+jobs:
+  bump_version:
+    name: Deploy 
+    runs-on: macos-10.15
+    defaults:
+      run:
+        working-directory: BuyPolish
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 
+          token: ${{ secrets.WORKFLOW_ACCESS_TOKEN_GITHUB }}
```

Comment:
Why do we need this access token/secret?

Reply:
This workflow will commit some changes and I want to trigger tests on that commit.